LayoutTestTests/SwiftLayoutTestCaseTests.swift

```diff
@@ -83,31 +83,31 @@ class SwiftTest: LayoutTestCase {
 
     func testBasicMoreGenericTestsWithLimitResultsCombinationsAndSizes() {
         var timesCalled = 0
-        runLayoutTestsWithViewProvider(NonViewCreationProtocol.self, limitResults: [.LimitDataCombinations, .NoSizes]) { (view: TestView, data, context) in
+        runLayoutTestsWithViewProvider(NonViewCreationProtocol.self, limitResults: [.limitDataCombinations, .noSizes]) { (view: TestView, data, context) in
             XCTAssertTrue(LYTIntegerValues().values.contains { $0 as? Int == context as? Int })
             timesCalled += 1;
         }
         XCTAssertEqual(timesCalled, polynomialCombinations)
     }
 
     class TestView: UIView, LYTViewProvider {
-        @objc static func dataSpecForTest() -> [NSObject : AnyObject] {
+        @objc static func dataSpecForTest() -> [AnyHashable: Any] {
```

Comment:
I think `AnyHashable` is more correct here because it gives you the freedom to do whatever you want with this. I don't want to limit it to JSON like data formats.